src/CalculatorUnitTests/DateCalculatorUnitTests.cpp

```diff
@@ -387,6 +387,65 @@ namespace DateCalculationUnitTests
             VERIFY_IS_TRUE(StringReference(L"") != viewModel->StrDateResult);
         }
 
+        TEST_METHOD(DateCalcViewModelDateDiffDaylightSavingTimeTest)
+        {
+          auto viewModel = ref new DateCalculatorViewModel();
+
+          viewModel->IsDateDiffMode = true;
+          VERIFY_IS_TRUE(viewModel->IsDateDiffMode);
+
+          /*29.02.2008*/
+          viewModel->FromDate = DateUtils::SystemTimeToDateTime(datetimeDifftest[5].startDate);
+          /*31.03.2008*/
+          viewModel->ToDate = DateUtils::SystemTimeToDateTime(datetimeDifftest[5].endDate);
+
+          //// Assert for the result
```

Comment:
I'm not sure about the style in calc, but I'd half expect this comment to be unnecessary (it should be obvious that VERIFY_... is asserting).